components/WorkDatesDisplay/WorkDatesDisplay.js

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from "react";
+import { DatesDisplay, DeleteButton, Dates } from "./WorkDatesDisplay.styled";
+
+export default function WorkDatesDisplay({ workDates, onDateRemove }) {
+  return (
+    <DatesDisplay>
+      <h4>Your Vacation Dates:</h4>
+      {workDates && workDates.length > 0 ? (
+        workDates.map((workDate) => (
```

Comment:
please always use a ul in combination with li elments as direct children if you map over an array.